contrib/face_recognition.lua

```diff
@@ -41,11 +41,21 @@ This plugin will add a new storage option and calls face_recognition after expor
 ]]
 
 local dt = require "darktable"
+local du = require "lib/dtutils"
 local df = require "lib/dtutils.file"
 local gettext = dt.gettext
 
--- works with darktable API version from 2.0.0 to 5.0.0
-dt.configuration.check_version(...,{2,0,0},{3,0,0},{4,0,0},{5,0,0})
+-- constants
+
+local MODULE = "face_recognition"
+local OUTPUT = dt.configuration.tmp_dir .. "/facerecognition.txt"
```

Comment:
Is the tmp dir the right one, or is the facerecognition.txt used longer?
Do we need the system independent path separator here?:
local PS = dt.configuration.running_os == "windows" and "\\" or "/"

Reply:
The facerecognition.txt file is used to output the results from face_recognition.  After it's populated, it's read to figure out what tags to apply.

When I read https://github.com/ageitgey/face_recognition/README.md it says linux and macos work and windows might, so I didn't work on adding windows compatibility.  I can go ahead and add it just in the face recognition software can be made to work on windows.  I'll add it and push it tonight.